src/scriptsys/lib_cmd.c

```diff
@@ -0,0 +1,53 @@
+#include "lua/lua.h"
```

Comment:
If you can come up with a short tag for script, I'd recommend having that at the beginning of functions and filenames.
Otherwise, two files with same name will cause a linker error, and two functions with same name will also fail to link.
I haven't been very diligent about this myself, other than with vkr, something I need to fix with existing code.

Reply:
Was wondering about this.